app/components/AuthenticatedRoute/index.jsx

```diff
@@ -48,14 +48,15 @@ const AuthenticatedRoute = ({
     <>
       <Route
         {...rest}
-        render={props => (<ChildComponent admin={admin} {...props} />)}
+        render={(props) => (<ChildComponent admin={admin} {...props} />)}
       />
     </>
   );
 };
 
 AuthenticatedRoute.propTypes = {
-  component: PropTypes.node.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
+  component: PropTypes.object.isRequired,
   // eslint-disable-next-line react/forbid-prop-types
   location: PropTypes.object,
```

Comment:
I thought we had defined a location propType before

Reply:
Not that I see, no

Reply:
Ohh, graphKB then ... out of context for this PR, but 

```
const LocationPropType = PropTypes.shape({
  hash: PropTypes.string,
  key: PropTypes.string,
  pathname: PropTypes.string,
  search: PropTypes.string,
});
```
is what's defined in graphKB